plone/app/widgets/dx.py

```diff
@@ -973,24 +962,6 @@ def RelatedItemsFieldWidget(field, request, extra=None):
     return FieldWidget(field, RelatedItemsWidget(request))
 
 
-if HAS_PAC:
-    @adapter(getSpecification(IDXCollection['query']), IFormLayer)
-    @implementer(IFieldWidget)
-    def QueryStringFieldWidget(field, request):
-        return FieldWidget(field, QueryStringWidget(request))
```

Comment:
Don't get me wrong. I never wanted to remove p.a.contenttypes support. I just want to make p.a.widgets experience for (addon-)developers easier than it is right now. An in case of RichWidget we can make it easier to add auto-support. I figure out your concerns about QueryWidget and if we don't get it for PAC without wiring in dx_bbb.py, I certainly leave this dependency untouched.